Libraries/Components/Intent/IntentAndroid.android.js

```diff
@@ -75,6 +77,16 @@ class IntentAndroid {
     IntentAndroidModule.canOpenURL(url, callback);
   }
 
+  /**
+   * If the app launch was triggered by an app link, it will pop the link url,
+   * otherwise it will return `null`
```

Comment:
Can you add a link to Android docs that explain how this works?

Thanks for making the API consistent with `LinkingIOS`!